Source/Painting/SvgColourConverter.cs

```diff
@@ -204,7 +204,30 @@ public override object ConvertTo(System.ComponentModel.ITypeDescriptorContext co
 #endif
             }
 
-            return base.ConvertTo(context, culture, value, destinationType);
+            return ToHtml((Color)value);

```

Comment:
Would it be better to fix `ConvertTo` to also handle the `NO_SDC` case?
This way it would could used elsewhere.

Reply:
Are you mean you want to change the `ConvertTo` method as follows?
```
        public override object ConvertTo(System.ComponentModel.ITypeDescriptorContext context, CultureInfo culture, object value, Type destinationType)
        {
            return ToHtml((Color)value);
        }
```

But the `ToHtml` method can not handle those `KnownColor`
For example, an `activeborder` color (comes from `Color.FromKnownColor(KnownColor.ActiveBorder)`), the `base.ConvertTo` and `ColorTranslator.ToHtml` return `activeborder`, `ToHtml` method will return a string start with `#`

Please confirm, thanks

Reply:
No, strike that, sorry - it should be ok as is.